tests/compile-fail/issue-3145.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("{}" a); //~ERROR expected token: `,`
```

Comment:
just make it a ui test. Rustc is moving away from compile-fail, there's no guarantee these are kept around.

Reply:
Done